astro.config.mjs

```diff
@@ -1,10 +1,12 @@
-import { defineConfig } from 'astro/config';
+import { defineConfig, passthroughImageService } from 'astro/config';
 import tailwind from '@astrojs/tailwind';
 
 // https://astro.build/config
 const config = {
-    integrations: [tailwind()],
-    base: '/claim'
+  integrations: [tailwind()],
+  image: {
+    service: passthroughImageService()
```

Comment:
`sharp` was giving me trouble in docker on m1 mac. I spent way too long troubleshooting it. This was the only way I could fix the issue... not ideal, turning off image optimization